### ISSUE TITLE
Extension causes tsserver to throw invalid syntax errors

### ISSUE THREAD

**Issue by iamfrontender** [42Crunch/vscode-openapi#23]
This extension makes tsserver throw invalid synax errors.
For the following environment:
```
Version: 1.38.1
Commit: b37e54c98e1a74ba89e03073e5a3761284e3ffb0
Date: 2019-09-11T13:30:08.229Z
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Linux x64 4.15.0-55-generic
```
And extension version:
```
Version 1.8.18 (October 8, 2019)
```

When enabling it, even simplest `.js` file causes `tsserver` to throw syntax errors. 

On screenshot below, clean install of VS code, with only this extension enabled (and its dependency)
![](https://img42.com/BGHui+)

Disabling the extension and reloading window removes the invalid errors.
![](https://img42.com/5TAoH+)

**Comment by ak1394** [comment]
Thanks for reporting! I couldn't reproduce it with 1.8.18 version of extension, but I suspect that this issue should be fixed in just-released version 1.8.19.

Could you update and see if it resolves it?

**Comment by iamfrontender** [comment]
Thanks for such a speedy reply!
After updating extension to `v1.8.19` it works flawlessly, hence closing this issue :)
